package.json

```diff
@@ -7,24 +7,25 @@
       "dist": "public"
     }
   },
-  "browserslist": [
-    "extends @wordpress/browserslist-config"
-  ],
-  "engines": {
-    "node": ">=16.0.0"
-  },
   "scripts": {
-    "dev": "bud dev",
     "build": "bud build",
+    "dev": "bud dev",
     "lint": "npm run lint:js && npm run lint:css",
-    "lint:js": "eslint resources/scripts",
     "lint:css": "stylelint \"resources/**/*.{css,scss,vue}\"",
+    "lint:js": "eslint resources/scripts",
     "test": "npm run lint"
   },
+  "browserslist": [
+    "extends @wordpress/browserslist-config"
+  ],
+  "resolutions": {
+    "globby": "^11.0.0"
+  },
   "devDependencies": {
     "@roots/bud": "^5.0.0",
     "@roots/sage": "^5.0.0",
     "@wordpress/browserslist-config": "4.1.0",
+    "@yarnpkg/plugin-dlx": "^3.1.1",
```

Comment:
This is only needed if you're going to interact with the API, it's not required to use `yarn dlx` through the CLI.